arches/management/commands/add_users.py

```diff
@@ -0,0 +1,186 @@
+"""
+ARCHES - a program developed to inventory and manage immovable cultural heritage.
+Copyright (C) 2013 J. Paul Getty Trust and World Monuments Fund
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as
+published by the Free Software Foundation, either version 3 of the
+License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+"""
+
+import argparse
+import csv
+import traceback
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+from django.contrib.auth.models import Group
+from django.contrib.auth.models import Permission
+from django.utils.crypto import get_random_string
+from arches.app.utils.permission_backend import assign_perm
+from arches.app.models import models
+
+
+class Command(BaseCommand):
+    """
+    Commands for adding arches test users
+
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "-o",
+            "--operation",
+            action="store",
+            dest="operation",
+            choices=[
+                "test_users",
+                "csv_users",
+            ],
+            default="test_users",
+            help="Operation Type; "
+            + "'test_users'=Creates test users"
+            + "'csv_users'=Loads users from CSV file",
+        )
+
+        parser.add_argument(
+            "-s",
+            "--source",
+            action="store",
+            dest="csv_file",
+            help="CSV file to be loaded",
+        )
+
+        parser.add_argument(
+            "-c",
+            "--count",
+            action="store",
+            dest="user_count",
+            default=3,
+            type=int,
+            help="The number of test users to add.  Not used for csv loading",
+        )
+
+        parser.add_argument(
+            "-p",
+            "--power-user",
+            action=argparse.BooleanOptionalAction,
+            help="Whether or not to create a power user",
+        )
+
+    def handle(self, *args, **options):
+        if options.get("operation") == "test_users":
+            self.add_test_users(options)
+        elif options.get("operation") == "csv_users":
+            self.load_users_from_csv(options)
+
+    def add_test_users(self, options):
+        user_count = options.get("user_count")
+        profiles = []
+        for current in range(1, user_count + 1):
+            profiles.append(
+                {
+                    "name": f"tester{current}",
+                    "email": f"tester{current}@test.com",
```

Comment:
If a user runs this command more than once because they will hit a unique constraint on the username and email fields because f"tester{current}" will always start at 1. To avoid that problem, I think you'd want to get the number of users already in they system and name the added users starting with that number. Something like this would work:
```suggestion
        user_count = options.get("user_count")
        profiles = []
        number_of_existing_users = User.objects.count()
        for current in range(1, user_count + 1):
            suffix = current + number_of_existing_users
            profiles.append(
                {
                    "name": f"tester{suffix}",
                    "email": f"tester{suffix}@test.com",
```